package.json

```diff
@@ -1,6 +1,6 @@
 {
-  "name": "@lampajr/bper",
-  "version": "3.1.1",
+  "name": "@kie/bper",
```

Comment:
wouldn't you call it just simply git-backporting? :thinking: 
```suggestion
  "name": "@kie/git-backporting",
```

Reply:
oh you are right, I missed to rename that - going to update README as well

Reply:
updated!